library/Exceptions/ValidationException.php

```diff
@@ -47,6 +47,28 @@ public static function stringify($value)
         }
     }
 
+    /**
+     * @param $value
+     * @return string
+     */
+    public static function stringifyArray($value)
+    {
+        $items = array();
+        foreach ($value as $val) {
+            if (is_object($val)) {
+                $items[] = self::stringifyObject($val);
+            } elseif (is_string($val)) {
+                $items[] = "'$val'";
+            } else {
+                $items[] = (string) $val;
```

Comment:
I don't think it's a good idea because maybe what we what is an array of arrays:

```php
v::in([
    [0,0],
    [0,1],
    [1,1],
    [1,0],
])->assert([1, 2]);
```

In this case the message will have `(Array, Array, Array, Array)`.

Some checking of recursion level is necessary, but I think it's also needed, since we're changing it now.

I think the best option is to use json_encode and make a normalization before. A good example is the [NormalizerFormatter](https://github.com/Seldaek/monolog/blob/master/src/Monolog/Formatter/NormalizerFormatter.php#L58-L139) from Monolog, but maybe I can implement that later, unless @nochso wants to do it.